snippets/ztp_TunedPerformancePatch.yaml

```diff
@@ -19,13 +18,12 @@ spec:
         kernel.timer_migration=1
```

Comment:
[bootloader]
        cmdline_crash=nohz_full=2-51,54-103
        
   This should be cleaned up from this example due to the changes in [OCPBUGS-6763](https://issues.redhat.com/browse/OCPBUGS-6763)

Reply:
Updated.